octobot_trading/orders/order_factory.py

```diff
@@ -0,0 +1,70 @@
+#  Drakkar-Software OctoBot-Trading
```

Comment:
It looks like methods from this file are not used yet, I guess it's for futur dependencies ?

Reply:
For example create_order_instance is used in TradingMode before calling trader.create_order.